<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotWebhooksDialog</class>
 <widget class="QDialog" name="BotWebhooksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Webhooks</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <widget class="QTableWidget" name="tableWebhooks">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>targetUrl</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>resource</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>event</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>targetUrl</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="leUrl"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnGetUrl">
            <property name="text">
             <string>Get Url</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>resource:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbResource">
            <item>
             <property name="text">
              <string>messages</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>memberships</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>rooms</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>event:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbEvent">
            <item>
             <property name="text">
              <string>created</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>updated</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>deleted</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnCreate">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnActive">
         <property name="text">
          <string>Active</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
